src/controllers/viewEvents.js

```diff
@@ -0,0 +1,11 @@
+const Events = require('../database/event_schema');
+const dbConnect = require('../database/db_connect');
+const viewQuery = require('../database/queries/view_events_querie');
+
+exports.get = (req, res, next) => {
+  viewQuery(undefined, (err, result) => {
+    // validate
+    if (err) return res.render('error', { error: 'abc', text: err });
```

Comment:
Don't render the error. It's a middleware that you access with ```next()```

Reply:
@FarahZaqout I have replaced it with next(err) :+1: